src/cli.rs

```diff
@@ -86,12 +89,19 @@ impl RunCommand for DataCommand {
     async fn run(&self, config: Config) -> Result<()> {
         info!("Running `data` subcommand");
 
+        let mut sp = Spinner::new(
+            DEFAULT_PROGRESS_SPINNER,
+            "Downloading and searching metadata from Azure".into(),
+        );
         let popgetter = Popgetter::new_with_config(config).await?;
+        let search_results = popgetter.search(self.search_params_args.clone().into());
 
-        let mut data = popgetter
-            .search(self.search_params_args.clone().into())
-            .download(&popgetter.config)
-            .await?;
+        sp.stop();
+        let len_requests = search_results.0.shape().0;
+        println!("Found {} metrics.", len_requests);
+        let mut sp = Spinner::new(DEFAULT_PROGRESS_SPINNER, "Download Parquet files".into());
```

Comment:
Perhaps reword this to:
```suggestion
        let mut sp = Spinner::new(DEFAULT_PROGRESS_SPINNER, "Downloading metrics".into());
```